### PR TITLE
Initial PLN objectives and quiz changes

### PR DESCRIPTION
**Pull request by daveo101**

### CODE REVIEW

**Comment by daveo101** [comment]
Not sure why the image won't load. It appears a previous image was coded but also didn't appear on live page.

**Comment by mrodriguez3313** [comment]
> Not sure why the image won't load. It appears a previous image was coded but also didn't appear on live page. 

No worries! For images to render like this you have to have the image and file that is referencing it in their own directory.

So what you need to do is make a new folder called (arbitrarily) "objectives". Then you need to move your file called "introduction" into it, but you HAVE to rename it to "index.md". Finally you have to move the image into the same directory and it will be able to render on the page.

**Comment by daveo101** [comment]
Hm... Tried that but still no luck. Also, now if I make any change on VS code it doesn't change on my localhost. Could that be due to moving and renaming the .md file?

**Review by walkerlj0 — CHANGES_REQUESTED** [review]
Hey, the quiz linked is still named 'PLN post-test' Change the name to PLN Quiz, and put all questions on to just one page, shouldn't be a two part thing!

File: content/en/curriculum/pln/pln-resources.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+        identifier: "pln-resources"
+weight: 62
+category: lecture
+level:
+- shallow
+- deep
+---
+## PLN Quiz
+
+Take the [**PLN Quiz**](https://docs.google.com/forms/d/e/1FAIpQLSfin7rDiiBqV9ybs1gV-n4hEv4bXmhanmf13Czd_zvDEHIsLw/viewform?usp=sf_link) after covering the pre-requisite content to gauge your understanding.
```
Comment (mrodriguez3313) [inline content/en/curriculum/pln/pln-resources.md:17]: I dont like the word pre-requisite here It makes it seem like they need to go through the "prerequisite" section of the curriculum as opposed to the stuff in this module. I would change it to be something like 
```suggestion
Take the [**PLN Quiz**](https://docs.google.com/forms/d/e/1FAIpQLSfin7rDiiBqV9ybs1gV-n4hEv4bXmhanmf13Czd_zvDEHIsLw/viewform?usp=sf_link) after covering the PLN module to gauge your understanding.
```

**Review by mrodriguez3313 — CHANGES_REQUESTED** [review]

**Pull request merged**